src/Microphysics/saturation_adjustment.jl

```diff
@@ -17,8 +20,6 @@ using DocStringExtensions: TYPEDSIGNATURES
 
 import ..Thermodynamics: saturation_specific_humidity
 
-abstract type AbstractEquilibrium end
-
```

Comment:
I'm assuming that this wasn't meant to be removed: it disappeared in a [merge commit](https://github.com/NumericalEarth/Breeze.jl/commit/4e642d144cee2203aa9c0a5f2662d0e106b12c7d), which feels unlikely to be done on purpose?